src/scss/style.scss

```diff
@@ -0,0 +1,17 @@
+@import "utils/mixin";
+@import "utils/media_query";
+@import "variable";
+
+@import "base/reset";
+@import "base/typhography";
+
+@import "./pages/home";
+@import "./pages/notFound";
+
+@import "components/input";
+@import "components/button";
+
+@import "layout/header";
+@import "layout/footer";
+
+@import "custom.scss";
```

Comment:
custom.scss file name can be improved

Reply:
This file is created to Override some styles in run time, as the project grows.